src/components/Data/Documents/Views/Column.vue

```diff
@@ -1,111 +1,132 @@
 <template>
-  <table class="centered highlight striped">
```

Comment:
REVIEWERS ! Don't look at this file, I'm changing it every days. It will be PR in few days...

Reply:
OK buddy